src/common/regions/regions_data.ts

```diff
@@ -152,6 +152,10 @@ export const statesByFips = fromPairs(
   states.map(state => [state.fipsCode, state]),
 );
 
+export const statesByStateCode = fromPairs(
+  states.map(state => [state.stateCode, state]),
+);
```

Comment:
I recently found about `_.keyBy`, it feels a bit more readable to me to create maps by a specific property

```ts
const statesByStateCode = keyBy(states, state => state.stateCode);
```

Reply:
SGTM!